packages/editor/src/ui-tools/tool-bar/tool-bar.css

```diff
@@ -1,12 +1,10 @@
 .ivy-tool-bar {
-  position: absolute;
   inset: 0;
   width: 100%;
   height: 48px;
   background: var(--glsp-tool-bar-bg);
   box-shadow: var(--glsp-box-shadow);
   z-index: 10;
-  container: toolbar / inline-size;
```

Comment:
why remove this?
without the container query at the bottom of this file will no longer work I think...

Reply:
Good point! I think I may have committed that by mistake. I'll push an update as soon as the other change is merged cause I'll need to rebase anyway, thank you!